src/main/webapp/yarn.lock

```diff
@@ -0,0 +1,9252 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Don't we already have a `src/main/webapp/yarn.lock` file. Why is this file being displayed as brand new? Maybe it's an artefact which requires merging `development` into this branch and the cart base branch?

Reply:
I actually do not have a good answer for you as to why this looks like a new file.  development and cart/_base both have it.  🤔 Very strange.

Reply:
It looks like the file `yarn.lock` does not exist in the branch **cart/_base** so I think in one of the previous pull requests it must have gotten deleted. In that case, it's okay to have it here (since this will just merge the file back in).